tests/schema/tests.py

```diff
@@ -1673,3 +1673,20 @@ def test_alter_field_default_dropped(self):
             )
             if connection.features.can_introspect_default:
                 self.assertIsNone(field.default)
+
+    def test_add_textfield_unhashable_default(self):
+        # Create the table
+        with connection.schema_editor() as editor:
+            editor.create_model(Author)
+        # Create a row
+        Author.objects.create(name='Anonymous1')
+        # Create a field that has an unhashable default
+        new_field = TextField(default={})
+        new_field.set_attributes_from_name("info")
+
+        def get_db_prep_save(default, connection):
+            return str(default)
+        new_field.get_db_prep_save = get_db_prep_save
+        # Fail if an exception is raised
+        with connection.schema_editor() as editor:
+            editor.add_field(Author, new_field)
```

Comment:
@MarkusH Cleaning up all the tests like this makes sense of course, but since this particular test is not yet committed and the change is rather trivial - it can be done before commit as well (and serve as an example of a good test later).

Besides, cleanup may happen but it might not as well for a plenty of reasons (no resources, unexpected difficulties, etc), so imo it's better to do this now with this particular test. Why should one commit something that already needs cleanup?

Hope this makes sense.